### PR TITLE
Replace MiniTK.Audio with OpenAL-CS and update to latest SDL2-CS.

### PR DESCRIPTION
**Pull request by pchote**
This removes our last hard dependency on the legacy OpenTK code.

I have forked https://github.com/OpenRA/OpenAL-CS because according to @Mailaender the upstream author does not want the library to be included in nuget.

### CODE REVIEW

**Comment by pchote** [comment]
Added a commit that switches us to the latest master branch of SDL2-CS.  The updated minitk-free dll is about 2.7MB smaller.

**Comment by Phrohdoh** [comment]
Code is good and sound works fine for me. :+1:
cc @Mailaender for Linux
cc @reaperrr for Windows.

**Comment by pchote** [comment]
Note to testers: be sure to do a make clean and delete anything in the thirdparty/download dir before doing a full rebuild.

**Comment by Mailaender** [comment]
Sound still works for me as well. :white_check_mark:

File: AUTHORS [hunk 1]
```diff
@@ -141,11 +141,8 @@ distributed under the CC BY-SA 3.0 license.
 Using SharpFont created by Robert Rouhani and
 distributed under the MIT license.
 
-Using the Open Toolkit distributed under the
-MIT license.
-
-Using SDL2# created by Ethan Lee and released
-under the zlib license.
+Using SDL2-CS and OpenAL-CS created by Ethan
+Lee and released under the zlib license.
```
Comment (penev92) [inline AUTHORS]: Can you break the above line *after* his name?

Comment (pchote) [inline AUTHORS]: No, not without breaking the rendering ingame.

**Comment by penev92** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/7252081c1ed78e444571fbf29d075524dcf5c763)

**Pull request merged**